### PR TITLE
Add Awesome Biomarkers List

### PR DESCRIPTION
**Pull request by markwk**
Hi, I created this list awhile back but forgot to contribute it back to the central list. The list is a collection of info on biomarkers and blood tests. Still a work in progress but key aspects are there now. Thanks for having me on the awesome list family!

**https://github.com/markwk/awesome-biomarkers**

The objective of the list is to help people to learn more about blood testing, biomarkers and tech and services to help quantify (and improve) your health!

I went through the requirements and added a TOC. I think everything else is covered. Ping me back if I missed something. Thanks!

### CODE REVIEW

**Comment by CompSciLauren** [comment]
I like the purpose behind this list a lot. I do think you should look over the requirements again though, it looks like some things are missing and there are a lot of awesome-lint errors. I think most of them should be pretty easy to fix, like some of them are from using "*" instead of "-" for formatting lists.

**Comment by markwk** [comment]
Thanks for taking a look @CompSciLauren and hat tip on awesome-linter since I had missed that. I correct everything except some odd error on the TOC that doesn't track with anything. Hopefully it's done or at least closer.

**Comment by CompSciLauren** [comment]
So that error is happening because you have nested sections but the TOC doesn't show them as being nested.

The preferred solution according to the awesome list requirements is to only have main sections and not include any subsections.
So you currently have subsections with three hashtags like this:
```
### Complete Blood Count Biomarkers
```
You need to remove a hashtag to make them not be subsections:
```
## Complete Blood Count Biomarkers
```

If you think it's necessary to have subsections, then those sections need to look nested in the TOC and you need to make sure to only include one level of nested-sections at most (one of your sections doesn't comply with this).

**Comment by markwk** [comment]
Ok. This list works best with multiple subsections so I'll leave that and wait for the lead maintainer to make the final call. Thanks. Appreciate the heads up.

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by krzjoa** [comment]
Awesome Lint returns:
```
  readme.md:11:1
  ✖  11:1  ToC item "Complete Blood Count Biomarkers" does not match corresponding heading "Blood Testing Companies"  remark-lint:awesome/toc

  1 error
```
However, I'm not sure what does it mean in this context.
Although guideline encourages us to make ToC as flat as possible, in this case I would use nested list with one indentation level or remove all the sublevels from the ToC. In my opinion, it'd be clearer :smiley:

**Comment by sindresorhus** [comment]
@markwk Ping

**Comment by markwk** [comment]
@sindresorhus I'm not sure what I'm missing besides 🦄 unicorn

File: readme.md [hunk 1]
```diff
@@ -635,6 +635,7 @@
 - [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.
 - [Connectivity Data and Reports](https://github.com/stevesong/awesome-connectivity-info#readme) - Better understand who has access to telecommunication and internet infrastructure and on what terms.
 - [Stacks](https://github.com/stackshareio/awesome-stacks#readme) - Tech stacks for building different apps and features.
+- [Biomarkers and Blood Tests](https://github.com/markwk/awesome-biomarkers) - Understand and track your blood testing and other lab results.
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
- [Biomarkers and Blood Tests](https://github.com/markwk/awesome-biomarkers#readme) - Understand and track your blood testing and other lab results. 
```

**Comment by sindresorhus** [comment]
The PR title is not correct. Please read the guidelines. They are carefully written so I don't have to point out obvious stuff.

---

The item title is `Biomarkers and Blood Tests`, while the heading in the list is `Awesome Biomarkers, Blood Tests and Health Indicators`. The heading in the list should be `Awesome Biomarkers and Blood Tests`.

---

I think the `Learn more about blood testing, biomarkers and tech and services to help quantify (and improve) your health!` could be more comprehensive.

---

It's weird to suddenly have a subheading called `Awesome List on Biomarkers` here: https://github.com/markwk/awesome-biomarkers#awesome-list-on-biomarkers

**Comment by sindresorhus** [comment]
Bump

**Pull request closed without merging**